.github/workflows/kvrocks.yaml

```diff
@@ -164,10 +174,8 @@ jobs:
 
       - name: Build Kvrocks
         run: |
-          ./x.py build -j$NPROC --unittest --compiler \
-            ${{ matrix.compiler }} ${{ matrix.with_ninja }} ${{ matrix.with_sanitizer }} \
-            ${{ matrix.without_jemalloc }} ${{ matrix.without_openssl }} ${{ matrix.without_luajit }} \
-            ${{ env.CMAKE_EXTRA_DEFS }}
+          ./x.py build -j$NPROC --unittest --compiler ${{ matrix.compiler }} ${{ matrix.without_jemalloc }} ${{ matrix.without_luajit }} \
```

Comment:
it seems that some parameters will be lost if there has an extra slash. For example, if `${{ env.CMAKE_EXTRA_DEFS }}` was empty and will become:
```shell
./x.py build -j8 --unittest --compiler \
  gcc --ninja \
  -DENABLE_OPENSSL=ON \
```
then it causes the TLS server to start failed. cc @PragmaTwice

Reply:
weird :rofl:

Reply:
😆 I tried to reorder parameters to work around, but didn't get what's the root cause. Really hope my conclusion is wrong.

Reply:
If it still not work well, I think we can write it in one line, no slash.

Reply:
ok~